code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -280,20 +280,6 @@
 				return
 	..()
 
-/mob/living/silicon/robot/drone/Bump(var/atom/movable/AM, yes)
```

Comment:
I'm protesting this being removed until it's readded in another way.

I'm also against people undoing nerfs just because it involves their pet mobs.